<launch>
	<arg name="input_point_topic" default="/points_raw" />
	<arg name="input_map_topic" default="/points_map" />
	<arg name="output_ground_topic" default="/points_ground" />
	<arg name="output_no_ground_topic" default="/points_no_ground" />

	<arg name="distance_threshold" default="0.3" />
	<arg name="min_clipping_height" default="0.0" />
	<arg name="max_clipping_height" default="100.0" />
	<arg name="min_clipping_range" default="5.0" />
	<arg name="max_clipping_range" default="50.0" />

	<node pkg="lidar_localizer" type="compare_map_reliability" name="compare_map_reliability">
		<remap from="/points_raw" to="$(arg input_point_topic)"/>
		<remap from="/points_map" to="$(arg input_map_topic)"/>
		<remap from="/points_ground" to="$(arg output_ground_topic)"/>
		<remap from="/points_no_ground" to="$(arg output_no_ground_topic)"/>

		<param name="distance_threshold" value="$(arg distance_threshold)" />
		<param name="min_clipping_height" value="$(arg min_clipping_height)" />
		<param name="max_clipping_height" value="$(arg max_clipping_height)" />
		<param name="min_clipping_range" value="$(arg min_clipping_range)" />
		<param name="max_clipping_range" value="$(arg max_clipping_range)" />
	</node>

</launch>
